virt/virt-4-11-release-notes.adoc

```diff
@@ -194,6 +194,22 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-11-known-issues"]
 == Known issues
 
+* Updating to {VirtProductName} {VirtVersion} from version 4.10 is blocked until you install the standalone Kubernetes NMState Operator. This occurs even if your cluster configuration does not use any link:https://nmstate.io/[nmstate] resources. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2126537[*BZ#2126537*])
+** As a workaround:
+. Verify that there are no node network configuration policies defined on the cluster:
++
+[source,terminal]
+----
+$ oc get nncp
+----
+. Add the following annotation to the `HyperConverged` custom resource (CR). The `HyperConverged` CR reports `TaintedConfiguration` condition when this patch is applied.
```

Comment:
The second sentence is unclear in its meaning, IMO (is this the desired result? is this just a note about a side effect? what does it do?). The bug makes it sound like it's a side effect.

I'm also wondering why we are using the `--overwrite` flag here by default. If we do use this flag, I think we should point out that it overwrites an existing jsonpatch (as mentioned in the bug doc text).

I might do something like:

```
** As a workaround:
. Verify that there are no node network configuration policies defined on the cluster:
+
[source,terminal]
----
$ oc get nncp
----

. If the list is empty, annotate the `HyperConverged` custom resource (CR).
.. If you do not have an existing JSON patch, overwrite the annotation by running the following command:
+
[source,terminal]
----
$ oc annotate --overwrite -n openshift-cnv hco kubevirt-hyperconverged 'networkaddonsconfigs.kubevirt.io/jsonpatch=[{"op": "replace","path": "/spec/nmstate", "value": null}]'
----
.. If you have an existing JSON patch, extend it:
+
[source,terminal]
----
// i don't know what the command should be
----

+
[NOTE]
====
The `HyperConverged` object reports a `TaintedConfiguration` condition while this patch is applied. This is benign.
====

. Update {VirtProductName}.

. After the update completes, remove the annotation by running the following command:
+
[source,terminal]
----
// i don't know what this command should be either
----
```

Reply:
> The second sentence is unclear in its meaning, IMO (is this the desired result? is this just a note about a side effect? what does it do?). The bug makes it sound like it's a side effect.

It is a side effect. Normally we want to notify customers that by setting the patch, they may have support issue. That does not apply here since we are the ones advising the change.

> .. If you have an existing JSON patch, extend it:

I'm not aware of a command that would extend the list in the patch itself. Perhaps we can use:

```
$ oc annotate --overwrite -n openshift-cnv hco kubevirt-hyperconverged 'networkaddonsconfigs.kubevirt.io/jsonpatch=[{"op": "replace","path": "/spec/nmstate", "value": null}, {other previously configured patches}, ...]'
```

? Asking the user to fill in the blanks.

> . After the update completes, remove the annotation by running the following command:

```
$ oc annotate -n openshift-cnv hco kubevirt-hyperconverged networkaddonsconfigs.kubevirt.io/jsonpatch-
```

Or they should set the original annotation back if there was any.